lib/model-builder.js

```diff
@@ -209,6 +211,11 @@ ModelBuilder.prototype.define = function defineClass(className, properties, sett
 
   util.inherits(ModelClass, ModelBaseClass);
 
+  // inherit validations
+  if (ModelBaseClass.validations && !settings.realmRequired) {
+    ModelClass.validations = traverse(ModelBaseClass.validations).clone();
```

Comment:
It's an object like this 
```js
{
  "name": [{
    "validation": "presence",
    "options": {}
  }]
}
```

i tried `ModelClass.validations = Object.create(ModelBaseClass.validations);` it creates a new clone, but 
because in [`isValid()`](https://github.com/strongloop/loopback-datasource-juggler/blob/89bf94245b32751422d7ff6d8432bd9c73c75b06/lib/validations.js#L450)
it does ` var attrs = Object.keys(validations || {});` so it wouldnt get the prototype keys it seems